chart/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
Nice, I didn't know it was capable of this.
But note that I didn't test this.

Reply:
@steven166 I included it because it is part of the helm create default template however, if the operator cannot handle HA/parallelism, might not be a good idea to include it.
Do you think it would work if there are multiple instance of the pod ? Or would it gives us undefined behavior ?

Reply:
I doubt if it will scale very well as it probably will run into concurrency issues and other bottlenecks first. I also don't think there are plenty of use cases where a single instance is not capable of handling all the work. So I think this HPA would be overkill for now.

Reply:
Thanks for the feedback! If there's concurrency issue, we have to remove the HPA. Will do that asap.